be/src/Project/Web/Buyer/Buyer.Web/Shared/Repositories/Clients/ClientsRepository.cs

```diff
@@ -36,17 +40,67 @@ public async Task<Client> GetClientAsync(string token, string language, Guid? id
 
             var response = await _apiClientService.GetAsync<ApiRequest<RequestModelBase>, RequestModelBase, Client>(apiRequest);
 
-            if (!response.IsSuccessStatusCode)
+            if (response.IsSuccessStatusCode is false)
             {
                 throw new CustomException(response.Message, (int)response.StatusCode);
             }
 
-            if (response.IsSuccessStatusCode && response.Data != null)
+            if (response.IsSuccessStatusCode && response.Data is not null)
             {
                 return response.Data;
             }
 
             return default;
         }
+
+        public async Task<List<ClientFieldValue>> GetClientFieldValuesAsync(string token, string language, Guid? id)
```

Comment:
suggestion: change to IEnumerable<ClientFieldValue> instead of List<> everywhere